src/main/java/com/google/api/codegen/go/GoContextCommon.java

```diff
@@ -34,4 +35,51 @@
     }
     return result;
   }
+
+  /**
+   * Converts and wraps the specified text into a comment block in the generated Go file.
+   *
+   * Lines are wrapped to 70 characters. See documentation of wrapLine for more details.
+   */
+  public Iterable<String> getCommentLinesWrap(String text) {
```

Comment:
How about calling this "getCommentLinesWrapped" or "getWrappedCommentLines"? I think these (especially the latter) reads better.